### PR TITLE
10.4.0 release notes

### PR DESCRIPTION
**Pull request by johanandren**
References #4170

### CODE REVIEW

**Comment by johanandren** [comment]
Failure is fossa

File: docs/src/main/paradox/release-notes/10.4.x.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# 10.4.x release notes
+
+## 10.4.0
+
+This release contains one small improvement and some dependency updates. It should be a drop in replacement
+for Akka HTTP 10.2.x.
```
Comment (patriknw) [inline docs/src/main/paradox/release-notes/10.4.x.md:6]: should we also mention the license change here? something similar to https://github.com/akka/akka/blob/main/akka-docs/src/main/paradox/project/migration-guide-2.6.x-2.7.x.md#migration-guide-26x-to-27x

Comment (patriknw) [reply docs/src/main/paradox/release-notes/10.4.x.md:6]: we should probably also mention that Akka 2.7.0 is required, dropping support for 2.6. and 2.5

Comment (johanandren) [reply docs/src/main/paradox/release-notes/10.4.x.md:6]: I was back and forth, wrote something about both those and then removed. I'll re-add.

Comment (johanandren) [reply docs/src/main/paradox/release-notes/10.4.x.md]: Trying to address in [ecafb91](https://github.com/akka/akka-http/pull/4171/commits/ecafb912617a35bbc4927bc0dfc85921ffe1eb77)

File: docs/src/main/paradox/release-notes/10.4.x.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+### Changes since 10.2.10
+
+#### akka-http-core
+
+* Update naming for status codes 413 and 422 [#4165](https://github.com/akka/akka-http/pull/4165)
+
+#### Dependency updates
+
+* Scala 2.12.17 & 2.13.10 [#4168](https://github.com/akka/akka-http/pull/4168)
+* Jackson Databind 2.13.4.10 [#4167](https://github.com/akka/akka-http/pull/4167)
```
Comment (patriknw) [inline docs/src/main/paradox/release-notes/10.4.x.md:22]:
```suggestion
* Jackson Databind 2.13.4.2 [#4172](https://github.com/akka/akka-http/pull/4172)
```

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**